src/components/SpeechBubble/index.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+import { css } from '@emotion/core'
+import { space, color, radius, fontSize, lineHeight } from '../../theme'
+
+const SpeechBubbleBottomContent = props => css`
+  margin-top: 10px;
+
+  &::before,
+  &::after {
+    top: -10px;
+  }
+
+  ${props.horizontalAlign === 'right' &&
+    css`
+      &::before {
+        border-bottom-right-radius: 40px 20px;
+        border-top-right-radius: 22px;
+      }
+      &::after {
+        border-bottom-right-radius: 100px 60px;
+      }
+    `};
+
+  ${props.horizontalAlign === 'left' &&
+    css`
+      &::before {
+        border-bottom-left-radius: 40px 20px;
+        border-top-left-radius: 22px;
+      }
+      &::after {
+        border-bottom-left-radius: 100px 60px;
+      }
+    `};
+`
+
+const SpeechBubbleTopContent = props => css`
+  margin-bottom: 10px;
+
+  &::before,
+  &::after {
+    bottom: -10px;
+  }
+
+  ${props.horizontalAlign === 'right' &&
+    css`
+      &::before {
+        border-top-right-radius: 40px 20px;
+        border-bottom-right-radius: 22px;
+      }
+      &::after {
+        border-top-right-radius: 100px 60px;
+      }
+    `};
+
+  ${props.horizontalAlign === 'left' &&
+    css`
+      &::before {
+        border-top-left-radius: 40px 20px;
+        border-bottom-left-radius: 22px;
+      }
+      &::after {
+        border-top-left-radius: 100px 60px;
+      }
+    `};
+`
+
+const SpeechBubbleContent = styled.div`
+  position: relative;
+  display: inline-block;
+  padding: ${space[16]};
+  color: ${color.space};
+  font-size: ${fontSize[16]};
+  line-height: ${lineHeight[1.25]};
```

Comment:
https://github.com/TicketSwap/solar/blob/master/src/theme.js#L160-L163
```suggestion
  line-height: ${lineHeight.title};
```